### DISCUSSION TITLE
Documentation for create-next-app templates

### DISCUSSION THREAD

**Discussion by pspeter3** [vercel/next.js#15016] [Help] [answered]
Is there documentation for creating a `create-next-app` template? I want to create a template for the side projects I work on which has Tailwind, TypeScript, ESLint, and Jest preconfigured.

**Comment by jamesmosier** [comment, marked as answer]
I don't think there is any particular documentation for creating a create-next-app template. In fact, the code for create-next-app just clones a repo you pass it, initializes a git repo, and installs packages from your package.json

**Comment by pspeter3** [comment]
Got it, thanks!

**Comment by Otoru** [comment]
how should i do so that my template repository receives the name of the project that i informed in `create-next-app`?
